include/deal.II/differentiation/sd/symengine_scalar_operations.h

```diff
@@ -0,0 +1,139 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_differentiation_sd_symengine_scalar_operations_h
+#define dealii_differentiation_sd_symengine_scalar_operations_h
+
+#include <deal.II/base/config.h>
+
+#ifdef DEAL_II_WITH_SYMENGINE
+
+#  include <deal.II/base/numbers.h>
+
+#  include <deal.II/differentiation/sd/symengine_number_types.h>
+#  include <deal.II/differentiation/sd/symengine_types.h>
+
+#  include <algorithm>
+#  include <type_traits>
+#  include <utility>
+#  include <vector>
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace Differentiation
+{
+  namespace SD
+  {
+    /**
+     * @name Symbolic variable creation
+     */
+    //@{
+
+    /**
+     * Return an Expression representing a scalar symbolic variable
+     * with the identifier specified by @p symbol.
+     *
+     * For example, if the @p symbol is the string <tt>"x"</tt> then the scalar
+     * symbolic variable that is returns represents the scalar $x$.
+     *
+     * @param[in] symbol An indentifier (or name) to the returned symbolic
+     *            variable.
+     * @return A scalar symbolic variable with the name @p symbol.
+     *
+     * @warning It is up to the user to ensure that there is no ambiguity in the
+     * symbols utilized within a section of code.
```

Comment:
I'd utilize the word `use` here, and everywhere below. :) 

> A number of critics have remarked that utilize is an unnecessary substitute for use. It is true that many occurrences of utilize could be replaced by use with no loss to anything but pretentiousness, for example, in sentences such as They utilized questionable methods in their analysis or We hope that many commuters will continue to utilize mass transit after the bridge has reopened. But utilize can mean "to find a profitable or practical use for."

Reply:
Fair enough :-D